assets/shared/styles/wp-components.scss

```
@@ -10,8 +10,24 @@
 
 body.sensei-color {
 
-	.components-button.is-primary.is-busy {
-		background-size: 100px 100%;
+	.components-button.is-primary {
+		&,
+		&:hover,
+		&:active,
+		&:focus,
+		&:disabled {
+			color: #000;
+		}
+		&:disabled {
+			background-color: rgba(70, 200, 173, 0.7);
```

This works with Sass variables too:
```suggestion
			background-color: rgba($primary, 0.7);
```